### DISCUSSION TITLE
Is there any way to use this library without a notification callback class?

### DISCUSSION THREAD

**Discussion by lopatoid** [Makuna/DFMiniMp3#70] [Q&A] [answered]
I don't need callback functions, so I wonder if there is a way to use this library without a notification callback class.

**Comment by Makuna** [comment, marked as answer]
The callback class is required.  
You can provide it an empty one (all the methods are empty).  The compiler should then optimize the code away completely.

Minimally most users will only be concerned with the error callback.  Since there can be transmission errors due to wiring; its a good practice to catch and handle that.
